viral/learning_stages.py

```
@@ -106,28 +137,52 @@ def get_session(
     return Cached2pSession.model_validate_json(path.read_text())
 
 
-def main() -> None:
-    mouse_name = "JB014"
+def get_mouse_sessions(mouse_name: str) -> Mouse2pSessions:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)